test/models/user_test.rb

```
@@ -0,0 +1,7 @@
+require "test_helper"
```

rubocop (1.69.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)